etc/travis/suites/application/install.sh

```diff
@@ -8,5 +8,10 @@ run_command "if [ ! -z \"${SYMFONY_VERSION}\" ]; then bin/require-symfony-versio
 run_command "composer install --no-interaction --prefer-dist" || exit $?
 run_command "composer dump-env test_cached" || exit $?
 
+print_header "Setting up JWT for API" "Sylius"
+run_command "source .env.test"
+run_command "openssl genrsa -out config/jwt/private-test.pem 4096 -algorithm rsa -passout env:JWT_PASSPHRAS rsa_keygen_bits:4096"
+run_command "openssl pkey -in config/jwt/private-test.pem -out config/jwt/public-test.pem -pubout"
```

Comment:
Should we expose this in our docs?

Reply:
Probably 👌

Reply:
But as we don't have any docs for the new API right now (instead of swagger which, maybe, is enough 💃) I would add it later 🚀

Reply:
That's one of the problems that people may have issues with running and testing it :( Let's move with this PR, but we need to provide this note in the installation guide.

Reply:
Both private-test.pem and public-test.pem (for test environment) could be included in the repository 
on .gitignore
!config/jwt/*-test.pem

Reply:
Hey Loic, could you open a PR with fix?

Reply:
@lchrusciel Yes here https://github.com/Sylius/Sylius/pull/11198 🥂